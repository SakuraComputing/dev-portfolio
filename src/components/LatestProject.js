import React from 'react';

export const LatestProject = () => (
    <section className="latest section">
        <div className="section-inner">
            <h2 className="heading">Latest Projects</h2>
            <div className="content">

                <div className="item featured text-center">
                    <h3 className="title"><a
                        href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-for-products-velocity/"
                        target="_blank">KickStarter - Velocity</a></h3>
                    <p className="summary">A responsive Bootstrap template designed for digital products</p>
                    <div className="featured-image">
                        <a href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-for-products-velocity/"
                           target="_blank">
                            <img className="img-responsive project-image"
                                 src="assets/images/projects/project-featured.png" alt="project name"/>
                        </a>
                        <div className="ribbon">
                            <div className="text">New</div>
                        </div>
                    </div>

                    <div className="desc text-left">
                        <p>You can promote your main project here. Suspendisse in tellus dolor. Vivamus a tortor eu
                            turpis pharetra consequat quis non metus. Aliquam aliquam, orci eu suscipit pellentesque,
                            mauris dui tincidunt enim, eget iaculis ante dolor non turpis.</p>
                        <p>Ut enim ad minima veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam,
                            nisi ut aliquid ex ea commodi consequatur. At vero eos et accusamus et iusto odio
                            dignissimos ducimus.</p>
                    </div>
                    <a className="btn btn-cta-secondary"
                       href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-for-products-velocity/"
                       target="_blank"><i className="fa fa-thumbs-o-up"></i> Back my project</a>
                </div>
                <hr className="divider"/>
                <div className="item row">
                    <a className="col-md-4 col-sm-4 col-xs-12"
                       href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-mobile-apps-atom/"
                       target="_blank">
                        <img className="img-responsive project-image" src="assets/images/projects/project-5.png"
                             alt="project name"/>
                    </a>
                    <div className="desc col-md-8 col-sm-8 col-xs-12">
                        <h3 className="title"><a
                            href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-mobile-apps-atom/"
                            target="_blank">Atom - Designed for Mobile Apps</a></h3>
                        <p>You can put one of your secondary projects here.</p>
                        <p><a className="more-link"
                              href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-mobile-apps-atom/"
                              target="_blank"><i className="fa fa-external-link"></i> Find out more</a></p>
                    </div>
                </div>
                <div className="item row">
                    <a className="col-md-4 col-sm-4 col-xs-12"
                       href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-web-development-agencies-devstudio/"
                       target="_blank">
                        <img className="img-responsive project-image" src="assets/images/projects/project-4.png"
                             alt="project name"/>
                    </a>
                    <div className="desc col-md-8 col-sm-8 col-xs-12">
                        <h3 className="title"><a
                            href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-web-development-agencies-devstudio/"
                            target="_blank">DevStudio - Designed for Web Development Agencies</a></h3>
                        <p>You can put one of your secondary projects here.</p>
                        <p><a className="more-link"
                              href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-web-development-agencies-devstudio/"
                              target="_blank"><i className="fa fa-external-link"></i> Find out more</a></p>
                    </div>
                </div>

                <div className="item row">
                    <a className="col-md-4 col-sm-4 col-xs-12"
                       href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-for-startups-tempo/"
                       target="_blank">
                        <img className="img-responsive project-image" src="assets/images/projects/project-1.png"
                             alt="project name"/>
                    </a>
                    <div className="desc col-md-8 col-sm-8 col-xs-12">
                        <h3 className="title"><a
                            href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-for-startups-tempo/"
                            target="_blank">Tempo - Designed for Startups</a></h3>
                        <p>You can put one of your secondary projects here.</p>
                        <p><a className="more-link"
                              href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-for-startups-tempo/"
                              target="_blank"><i className="fa fa-external-link"></i> Find out more</a></p>
                    </div>
                </div>
                <div className="item row">
                    <a className="col-md-4 col-sm-4 col-xs-12"
                       href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-for-mobile-apps-delta/"
                       target="_blank">
                        <img className="img-responsive project-image" src="assets/images/projects/project-2.png"
                             alt="project name"/>
                    </a>
                    <div className="desc col-md-8 col-sm-8 col-xs-12">
                        <h3 className="title"><a
                            href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-for-mobile-apps-delta/"
                            target="_blank">Delta - Designed for Mobile Apps</a></h3>
                        <p> You can put one of your secondary projects here.</p>
                        <p><a className="more-link"
                              href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-for-mobile-apps-delta/"
                              target="_blank"><i className="fa fa-external-link"></i> Find out more</a></p>
                    </div>
                </div>
            </div>

        </div>
    </section>

);

export default LatestProject;