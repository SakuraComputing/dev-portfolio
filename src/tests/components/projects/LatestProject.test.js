import React from 'react';
import { shallow } from 'enzyme';
import LatestProject from '../../../components/projects/LatestProject';

describe('Latest Project Section', () => {
    it('should render correctly', () => {
        const wrapper = shallow(<LatestProject />);
        expect(wrapper).toMatchSnapshot();
    });
});