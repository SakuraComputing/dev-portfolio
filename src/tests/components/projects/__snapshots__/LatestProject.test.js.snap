// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Latest Project Section should render correctly 1`] = `
<section
  className="latest section"
>
  <div
    className="section-inner"
  >
    <h2
      className="heading"
    >
      Latest Projects
    </h2>
    <div
      className="content"
    >
      <div
        className="item featured text-center"
      >
        <h3
          className="title"
        >
          <a
            href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-for-products-velocity/"
            target="_blank"
          >
            KickStarter - Velocity
          </a>
        </h3>
        <p
          className="summary"
        >
          A responsive Bootstrap template designed for digital products
        </p>
        <div
          className="featured-image"
        >
          <a
            href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-for-products-velocity/"
            target="_blank"
          >
            <img
              alt="project name"
              className="img-responsive project-image"
              src="assets/images/projects/project-featured.png"
            />
          </a>
          <div
            className="ribbon"
          >
            <div
              className="text"
            >
              New
            </div>
          </div>
        </div>
        <div
          className="desc text-left"
        >
          <p>
            You can promote your main project here. Suspendisse in tellus dolor. Vivamus a tortor eu turpis pharetra consequat quis non metus. Aliquam aliquam, orci eu suscipit pellentesque, mauris dui tincidunt enim, eget iaculis ante dolor non turpis.
          </p>
          <p>
            Ut enim ad minima veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur. At vero eos et accusamus et iusto odio dignissimos ducimus.
          </p>
        </div>
        <a
          className="btn btn-cta-secondary"
          href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-for-products-velocity/"
          target="_blank"
        >
          <i
            className="fa fa-thumbs-o-up"
          />
           Back my project
        </a>
      </div>
      <hr
        className="divider"
      />
      <div
        className="item row"
      >
        <a
          className="col-md-4 col-sm-4 col-xs-12"
          href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-mobile-apps-atom/"
          target="_blank"
        >
          <img
            alt="project name"
            className="img-responsive project-image"
            src="assets/images/projects/project-5.png"
          />
        </a>
        <div
          className="desc col-md-8 col-sm-8 col-xs-12"
        >
          <h3
            className="title"
          >
            <a
              href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-mobile-apps-atom/"
              target="_blank"
            >
              Atom - Designed for Mobile Apps
            </a>
          </h3>
          <p>
            You can put one of your secondary projects here.
          </p>
          <p>
            <a
              className="more-link"
              href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-mobile-apps-atom/"
              target="_blank"
            >
              <i
                className="fa fa-external-link"
              />
               Find out more
            </a>
          </p>
        </div>
      </div>
      <div
        className="item row"
      >
        <a
          className="col-md-4 col-sm-4 col-xs-12"
          href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-web-development-agencies-devstudio/"
          target="_blank"
        >
          <img
            alt="project name"
            className="img-responsive project-image"
            src="assets/images/projects/project-4.png"
          />
        </a>
        <div
          className="desc col-md-8 col-sm-8 col-xs-12"
        >
          <h3
            className="title"
          >
            <a
              href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-web-development-agencies-devstudio/"
              target="_blank"
            >
              DevStudio - Designed for Web Development Agencies
            </a>
          </h3>
          <p>
            You can put one of your secondary projects here.
          </p>
          <p>
            <a
              className="more-link"
              href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-web-development-agencies-devstudio/"
              target="_blank"
            >
              <i
                className="fa fa-external-link"
              />
               Find out more
            </a>
          </p>
        </div>
      </div>
      <div
        className="item row"
      >
        <a
          className="col-md-4 col-sm-4 col-xs-12"
          href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-for-startups-tempo/"
          target="_blank"
        >
          <img
            alt="project name"
            className="img-responsive project-image"
            src="assets/images/projects/project-1.png"
          />
        </a>
        <div
          className="desc col-md-8 col-sm-8 col-xs-12"
        >
          <h3
            className="title"
          >
            <a
              href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-for-startups-tempo/"
              target="_blank"
            >
              Tempo - Designed for Startups
            </a>
          </h3>
          <p>
            You can put one of your secondary projects here.
          </p>
          <p>
            <a
              className="more-link"
              href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-for-startups-tempo/"
              target="_blank"
            >
              <i
                className="fa fa-external-link"
              />
               Find out more
            </a>
          </p>
        </div>
      </div>
      <div
        className="item row"
      >
        <a
          className="col-md-4 col-sm-4 col-xs-12"
          href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-for-mobile-apps-delta/"
          target="_blank"
        >
          <img
            alt="project name"
            className="img-responsive project-image"
            src="assets/images/projects/project-2.png"
          />
        </a>
        <div
          className="desc col-md-8 col-sm-8 col-xs-12"
        >
          <h3
            className="title"
          >
            <a
              href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-for-mobile-apps-delta/"
              target="_blank"
            >
              Delta - Designed for Mobile Apps
            </a>
          </h3>
          <p>
             You can put one of your secondary projects here.
          </p>
          <p>
            <a
              className="more-link"
              href="http://themes.3rdwavemedia.com/website-templates/responsive-bootstrap-theme-for-mobile-apps-delta/"
              target="_blank"
            >
              <i
                className="fa fa-external-link"
              />
               Find out more
            </a>
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
`;
